src/components/interfaces/MOBILE_API.xml

```diff
@@ -6345,6 +6345,13 @@
 
     <function name="OnRCStatus" functionID="OnRCStatusID" messagetype="notification">
       <description>Issued by SDL to notify the application about remote control status change on SDL</description>
+      <param name="allowed" type="Boolean" mandatory="false">
+        <description>
+            If "true" - RC is allowed; if "false" - RC is disallowed.
+            Not present in notification in case by module allocation/deallocation.
```

Comment:
This description is incorrect. 

Please use description used in the evolution [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0172-onRcStatus-allowed.md)

> <description>If "true" - RC is allowed; if "false" - RC is disallowed.</description>

Reply:
@JackLivio Fixed 
8cfa86d